blocks/header/header.js

```diff
@@ -103,10 +119,12 @@ export default async function decorate(block) {
     nav.id = 'nav';
     nav.innerHTML = html;
 
-    const classes = ['brand', 'sections', 'tools'];
```

Comment:
Decided to switch to dynamically injecting the logo element to make the DOM  match a little closer to the clients original implementation and simplify the layout.